website/docs/docs/dbt-cloud/access-control/access-control-overview.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Access Control Overview"
+id: "access-control-overview"
+---
+
+## Overview
+This overview provides a basic explanation of access control in dbt Cloud.
+
+dbt Cloud approaches access control by combining aspects from both of the following models:
+
+- **Seat-based Access Control (LAC):** Each user is associated with a seat license, which allows for specific capabilities. 
+- **Role-based Access Control (RBAC):** Access privileges are assigned to groups which in term are assigned to users. 
```

Comment:
Similar thought here: do we want to say what RBAC is rather than describe the way it is implemented in this section?

Reply:
@drewbanin What do you think of this?
```
RBAC is a method of restricting access to dbt Cloud objects based on the access privileges assigned to groups which in term are assigned to users. This can be used to control what a user can access both on the account and dbt Cloud project level.
```